tools/python/test_server/start_server.py

```diff
@@ -1,7 +1,19 @@
 #!/usr/bin/env python3
 
+"""
+    Run from `$OMIM_ROOT/tools/python/test_server` dir
+"""
+
 import subprocess
 
 SERVER_RUNNABLE = 'server/testserver.py'
+LOG_FILE = 'server.log'
+
+with open(LOG_FILE, 'w') as log:
+    subprocess.Popen(['python3', SERVER_RUNNABLE], stdout=subprocess.DEVNULL, stderr=log, text=True)
 
-subprocess.Popen(['python3', SERVER_RUNNABLE], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+while True:
+    with open(LOG_FILE, 'r') as log:
+        data = log.read()
```

Comment:
Why not readline?

Reply:
Because it's a universal solution.
If you run this script manually several times without stopping the server, the line with "Started server with pid" may not be the last one.
Plus when you write to the file and read from it asynchronously, there may be artifacts while reading only one line.

I've already tried using readline and encountered this behavior.

Reply:
Should it be
```
with open(LOG_FILE, 'r') as log:
    while True:
        data = log.readline()
```
instead?

Reply:
No